doc/rewriting-services.md

```diff
@@ -1,288 +0,0 @@
-# Tips for rewriting legacy services
```

Comment:
This document has a lot of good content related to best practices for new-style services. Rather than delete it, I'd like to rework it into a design document.

Could you reset this file and then we can deal with that in a follow on?

If you want you could add a note at the top that the document is out of date and needs to be updated, so people take it with a grain of salt.

Reply:
Done.